adapters/madvertise/madvertisetest/exemplary/simple-video.json

```diff
@@ -0,0 +1,254 @@
+                        }
+                    },
+                    "app": {
+                        "id": "9561622",
+                        "bundle": "com.madvertise.bluestack",
+                        "name": "FR_BlueStack_App_Android_MNG",
+                        "storeurl": "https://play.google.com/store/apps/details?id=com.madvertise.bluestack",
+                        "cat": [
+                            "IAB6",
+                            "IAB9",
+                            "IAB10",
+                            "IAB12",
+                            "IAB14",
+                            "IAB20"
+                        ]
+                    },
+                    "device": {
+                        "ip": "123.123.123.123",
+                        "ua": "test-user-agent",
+                        "dnt": 0,
+                        "lmt": 0,
+                        "ifa": "b879a967-cccc-bbbb-aaaa-1aa111111111",
+                        "make": "Sony",
+                        "model": "G8441",
+                        "os": "Android",
+                        "osv": "9.0",
+                        "devicetype": 4,
+                        "connectiontype": 6,
+                        "js": 1,
+                        "carrier": "Free",
+                        "geo": {
+                            "type": 2,
+                            "country": "FRA"
+                        }
+                    },
+                    "imp": [
+                        {
+                            "id": "test-imp-id",
+                            "secure": 1,
+                            "bidfloor": 3,
+                            "bidfloorcur": "EUR",
+                            "video": {
+                                "mimes": [
+                                    "video/mp4"
+                                ],
+                                "w": 320,
+                                "h": 480,
+                                "minduration": 2,
+                                "maxduration": 30,
+                                "playbackmethod": [
+                                    1,
+                                    3
+                                ],
+                                "protocols": [
+                                    1,
+                                    2,
+                                    3,
+                                    4,
+                                    5,
+                                    6
+                                ],
+                                "placement": 5
+                            },
+                            "ext": {
+                                "bidder": {
+                                    "zoneId": "/1111111/banner"
```

Comment:
Should the value of `zoneId` here be `/1111111/video` since this is a video example?

Reply:
Yes. I push update ([master 18d0fd63] video zoneId)